lib/google_maps.ex

```diff
@@ -851,6 +851,112 @@ defmodule GoogleMaps do
     GoogleMaps.get("place/queryautocomplete", params)
   end
 
+@doc """
+  Search for nearby places based on location and keywords
+
+  `nearby_search/3/4` - Most common use for Places API.
+
+  `nearby_search/1/2` - Intended for use with `rankby` 
+    when you cant use radius.
+
+## Args:
+
+    * `location` — The latitude/longitude around which to
```

Comment:
While Google only takes `"latitude,longitude"` stirng as argument, we can support the `coordinate` type defined at the top. The function will adapt accordingly.